readme.md

```diff
@@ -524,6 +524,53 @@ test.only.serial(...);
 
 This means you can temporarily add `.skip` or `.only` at the end of a test or hook definition without having to make any other changes.
 
+### Test macros
+
+Additional arguments passed to the test declaration will be passed to the test implementation. This is useful for creating reusable test macros.
```

Comment:
Should we maybe mention why it's beneficial to use test macros over hand-made loops/etc? (To make static analysis possible === better perf).